hbase-server/src/main/java/org/apache/hadoop/hbase/master/snapshot/SnapshotFileCache.java

```diff
@@ -97,6 +98,8 @@ Collection<String> filesUnderSnapshot(final FileSystem fs, final Path snapshotDi
   private ImmutableMap<String, SnapshotDirectoryInfo> snapshots = ImmutableMap.of();
   private final Timer refreshTimer;
 
+  private static final int LOCK_TIMEOUT_MS = 30000;
```

Comment:
Should we make it configurable? Just asking, I'm neutral on whether to introduce a configuration.

Reply:
After HBASE-27590 I don't think it is necessary because the locking time is drastically reduced. Earlier I have seen the `Failed to acquire write lock on taking snapshot after waiting 60000ms` WARN messages continuously on this cluster.